build.sbt

```diff
@@ -40,7 +40,7 @@ lazy val standardSettings = Defaults.defaultSettings ++ Seq(
     "-language:higherKinds",
     "-language:implicitConversions",
     "-unchecked",
-    "-Xfatal-warnings",
+    //"-Xfatal-warnings", temporarly removed in order for the whole thing to build. If we could change this setting to be selectively applied in the build script, then it might make sense to remove it permanently here, anyway...
```

Comment:
Ok, I guess as long as everything builds cleanly and all tests pass, though it might be convenient to have the old tests around while porting them (or are they all done now?).